### PR TITLE
[k8s] Manage general resources with terraform

### PR DESCRIPTION
**Pull request by olivertso**

### CODE REVIEW

**Comment by olivertso** [comment]
atlantis plan

**Comment by olivertso** [comment]
atlantis plan

File: terraform/general/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module "ecr" {
+  source = "git@github.com:loadsmart/terraform-modules.git//aws-ecr"
+
+  project = "${local.project}"
+  build   = true
```
Comment (eduardocardoso) [inline terraform/general/ecr.tf]: The build is not working correctly, remember? You can leave this on, but keep in mind that the image won't work.

Comment (eduardocardoso) [reply terraform/general/ecr.tf]: Also, this repo doesn't have a Dockerfile, the build will fail anyway

Comment (olivertso) [reply terraform/general/ecr.tf]: Yes I remember, but forgot to change it to `false`, thanks for reminding.
The project has a Dockerfile in the root directory, I'll put it into a `docker` folder.

**Pull request merged**